libs/query-engine-common/Cargo.toml

```diff
@@ -20,11 +20,11 @@ tracing.workspace = true
 tracing-subscriber = { version = "0.3" }
 tracing-futures = "0.2"
 tracing-opentelemetry = "0.17.3"
-opentelemetry = { version = "0.17"}
+opentelemetry = { version = "0.17" }
```

Comment:
Automatic formatting by VSCode I can revert it if necessary

Reply:
That should be ok to keep 👍🏼